src/main/java/com/example/petbuddybackend/service/chat/ChatService.java

```diff
@@ -61,14 +64,19 @@ public Page<ChatMessageDTO> getChatMessagesByParticipantEmail(Long chatId,
     }
 
     @Transactional(readOnly = true)
-    public Page<ChatRoomDTO> getChatRoomsByParticipantEmailSortedByLastMessage(String principalEmail,
-                                                                               Role role,
-                                                                               Pageable pageable,
-                                                                               ZoneId timeZone) {
-        Page<ChatMessage> principalChatMessages = role == Role.CLIENT ?
-                chatMessageRepository.findLastMessagesOfChatRoomsOfClient(principalEmail, pageable) :
-                chatMessageRepository.findLastMessagesOfChatRoomsOfCaretaker(principalEmail, pageable);
+    public Page<ChatRoomDTO> getChatRoomsByCriteriaSortedByLastMessage(String principalEmail,
```

Comment:
```suggestion
    public Page<ChatRoomDTO> getChatRooms(String principalEmail,
```

nwm czy bym tego tak nie zostawil

Reply:
Inne metodki z tej klasy też mniej więcej na podobną konwencję pozmieniałem w razie czego